src/static/helpers/slasHelper.test.ts

```diff
@@ -232,18 +232,13 @@ describe('Authorize user', () => {
     await slasHelper.authorize(mockSlasClient, codeVerifier, parameters, true);
 
     // There should be no code_challenge for private client
-    const expectedReqOptions = {
-      accessToken: "access_token",
-      client_id: 'client_id',
-      channel_id: 'site_id',
-      dnt: "false",
-      hint: 'hint',
-      redirect_uri: 'redirect_uri',
-      refreshToken: 'refresh_token',
-      response_type: 'code',
-      usid: 'usid',
+    const unexpectedQueryParams = {
```

Comment:
Dumb question, why do we need to change this test?

Reply:
This test fix addresses a change that was made to for allowing any query parameters through: https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic/pull/196

Rather than check for expected options, I think it's better we assert that the code challenge wasn't included. That way, if new non-breaking options appear in the future, this test will continue to work.